news/2020-08-03-mappers-report-july.md

```diff
@@ -0,0 +1,183 @@
+We've got your interest? Great! When you are sure that the activity lenience can be applied to you, contact any of the Nomination Assessment Team members! (Preferably of your game mode, because they will most likely know if you were a Beatmap Nominator in the past.)
+
+### BN application and evaluation results will now be sent via chat
+
+As you may already know, all messages regarding Beatmap Nominator applications and evaluations results used to be delivered through legacy pm system on the old osu! website. **This is no longer the case**. You will now receive a message through the website/in-game chat function once your application or evaluation has concluded. This message will contain a link that will lead you to the **Beatmap Nominator website**, where more detailed feedback can be viewed. 
+
+And now let's move on to the team movements for this month:
+
+### BN Additions
+
+- **osu!:** [-Aqua](https://osu.ppy.sh/users/7150015), [Bibbity Bill](https://osu.ppy.sh/users/4446810), [Log Off Now](https://osu.ppy.sh/users/4378277), [C00L](https://osu.ppy.sh/users/4930630), [Ayucchi](https://osu.ppy.sh/users/7823498) & [Hobbes2](https://osu.ppy.sh/users/8157492)
+- **osu!catch:** [wonjae](https://osu.ppy.sh/users/5032045) & [-Joni-](https://osu.ppy.sh/users/9988837)
+- **osu!mania:** [Arzenvald](https://osu.ppy.sh/users/3027421), [lenpai](https://osu.ppy.sh/users/5314573) & [Davvy](https://osu.ppy.sh/users/10047413)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. 
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!:** [Peter](https://osu.ppy.sh/users/8623835), [Kaitjuh](https://osu.ppy.sh/users/2225327), [Icekalt](https://osu.ppy.sh/users/5410645) & [Rizen](https://osu.ppy.sh/users/3056732)
+- **osu!taiko:** [Volta](https://osu.ppy.sh/users/4154071)
+- **osu!catch:** [JBHyperion](https://osu.ppy.sh/users/4879508)
+
+### NAT Additions 
+
+A new member in the Nomination Assessment has risen! Please welcome [Scotty](https://osu.ppy.sh/users/11085809) to the team! After around one and half a year as a Beatmap Nominator, he will now help the current osu!mania NAT with the Beatmap Nominator evaluations and make sure that the recent flow of new members will not break up. 
+Welcome aboard Scotty!
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+Another month, another Mappers' Showcase! However, the term "mapper" is maybe not the right one this time. Since the showcase has covered all game modes at least once, we thought we'd make something special this time. We would like to show you the work of storyboarders!
+As storyboarding can apply to all game modes, we thought it'd be the perfect moment to showcase some outstanding storyboarders this month!
```

Comment:
```suggestion
Another month, another Mappers' Showcase! However, the term "mapper" is maybe not the right one this time. Since the showcase has covered all game modes at least once, we thought we'd make something special this time. We would like to show you the work of storyboarders! As storyboarding can apply to all game modes, we thought it'd be the perfect moment to showcase some outstanding storyboarders this month!
```